apps/web/components/ContactInformation/ContactInformationForm.vue

```diff
@@ -1,19 +1,19 @@
 <template>
-  <form data-testid="contact-information-form" @submit.prevent="$emit('on-save', cart.customerEmail)">
+  <form data-testid="contact-information-form" @submit.prevent="onSubmit" novalidate>
     <label>
       <UiFormLabel>{{ $t('contactInfo.email') }}</UiFormLabel>
-      <SfInput name="email" type="email" v-model="cart.customerEmail" required />
+      <InputText name="email" type="email" />
```

Comment:
Name needs a unique name/id otherwise it could conflict with other forms that use the useForm